code/modules/research/designs/medical_designs.dm

```diff
@@ -460,6 +460,17 @@
 	build_path = /obj/item/organ/cyberimp/chest/thrusters
 	category = list("Misc", "Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+	
+/datum/design/clownservo
+	name = "Snappop Waddle Servo"
+	desc = "An alleged violation of clown rights"
+	id = "clownservo"
+	build_type = PROTOLATHE | MECHFAB
+	construction_time = 40
+	materials = list(/datum/material/iron = 500, /datum/material/glass = 500)
```

Comment:
since this is clown-related, I kind of feel like there should be some bananium involved here

but I'm totally cool with it just costing iron and glass, since I doubt that this'd ever get printed if it cost something as rare and precious as bananium (and could be mimicked by just turning on the waddling of your clown shoes)